core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -168,7 +168,7 @@ public boolean inodeIdExists(long id) {
   public boolean inodePathExists(AlluxioURI path) {
```

Comment:
no, the type is already an AlluxioURI, I just want to change the variable name from `path` to `uri`